aircon/aircon.py

```diff
@@ -180,8 +181,11 @@ def _build_command(self, name: str, data_value: int):
   def _convert_to_control_value(self, name: str, value) -> int:
     raise NotImplementedError()
 
-  def queue_status(self) -> None:
+  async def queue_status(self) -> None:
+    _WAIT_FOR_EMPTY_QUEUE = 1
     for data_field in fields(self._properties):
+      while self.commands_queue.qsize() > 0:
```

Comment:
That was the first issue I tried to solve.
When that busy loops run, you **can't** issue any command. And that happens every 10 minutes.
The solution I had, gave priority to other commands than the status.
PriorityQueue, I don't familiar with... It just has to give more priority to commands other than the status update.
You will still have to control the order of the commands.

Reply:
Right, so the idea is to use `queue.PriorityQueue` instead of `queue.Queue`, and set the priority to e.g. 10 for "normal" commands, and 100 for these status updates. Then any normal command will be prioritized over the status updates.
The queue orders the elements by a using `__lt__`, which is implemented by `dataclass` in field order (skipping fields with `compare=False`) - so just setting the correct values will do the right thing.

Reply:
Can you please merge this and change it? I think you're gonna handle this better and faster than me :) Meantime I try to investigate another issue I see with some of the AC which does not report the correct value.

Reply:
Sure, I can do it. but let's revert this change.

Reply:
Ok, reverted.

Reply:
Thanks. I've submitted 834fab2 for this, you can try it out - didn't get a chance to test it yet.

Reply:
Thanks :) it works as expected 👍